index.html

```diff
@@ -153,31 +170,33 @@
 			"lines": [
 				// Quận 3
 				//Example
-				// {
-				// 	type: "polyline",
-				// 	paths: [
-				// 		[106.79134386128325, 10.866997521468168],
-				// 		[106.79413335865881, 10.865395978158253],
-				// 		[106.79833906239429, 10.864637349380688],
-				// 		[106.80366056507997, 10.867882581189418],
-				// 		[106.8083812529463, 10.873403608756607],
-				// 		[106.80511968678411, 10.875552988801912],
-				// 		[106.80318849629333, 10.876522311989321],
-				// 		[106.80074232167169, 10.876522311989321],
-				// 		[106.79949777668874, 10.878798101871515],
-				// 		[106.79675119465743, 10.878882390052073],
-				// 		[106.78988473957912, 10.875637277899806],
-				// 		[106.79134386128325, 10.866997521468168]
-				// 	],
-				// 	symbol: {
-				// 		type: "simple-line",
-				// 		color: [226, 119, 40], // orange
-				// 		width: 2
-				// 	},
-				// 	Name: "Làng đại học",
-				// 	Location: "Quận Thủ Đức",
-				// 	popupTemplate: point_template_area
-				// }
+				{
+					type: "polyline",
+					paths: [
+
+						//Phường 4
+						[106.68209335660033, 10.770074710273892],
+						[106.67904430070908, 10.772709893579371], 
+						[106.68282879153882, 10.775936355835066], 
+						[106.68271611794627, 10.775984486566879], 
+						[106.68265659836406, 10.776052516972653], 
+						[106.68175874717366, 10.777547601177377], 
+						[106.68184727867835, 10.777598757636863], 
+						[106.68192087832313, 10.777675073803865], 
+						[106.68638256253398, 10.775275491824537], 
+						[106.68682577057525, 10.775030447952853], 
+						[106.68481506507447, 10.772919119592833], 
+						[106.68209335660033, 10.770074710273892]
+					],
+					symbol: {
+						type: "simple-line",
+						color: [226, 119, 40], // orange
+						width: 2
+					},
+					Name: "Quận 3",
+					Location: "Quận Thủ Đức",
+					popupTemplate: point_template_area

```

Comment:
Move it to another file. Remove your comment. And what mean of Name: Quận 3 while Location: Quận Thủ Đức?

Reply:
removed my comment  and changed Location to Quận 3